workflow/scripts/plot_ariadne_variables.py

```diff
@@ -7,6 +7,75 @@
 
 EUR20TOEUR23 = 1.1076
 
+def plot_Kernnetz(df, savepath=None):
+    key = "Investment|Energy Supply|Hydrogen|Transmission and Distribution|"
+
+    data = {
+        "Kategorie": ["FNB", "PyPSA"],
+        "Kernnetz-Zubau": [
+            12.3+0.6,
```

Comment:
pypsa costs are in EUR2020, Kernnetz costs probably in EUR23 (?)

Make sure to apply an inflation correction (see e.g. the plot_NEP function)